tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -4,107 +4,80 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 #include <zephyr/drivers/dac.h>
 #include <zephyr/kernel.h>
 #include <zephyr/ztest.h>
 
-#if defined(CONFIG_BOARD_NUCLEO_F091RC) || \
-	defined(CONFIG_BOARD_NUCLEO_F207ZG) || \
-	defined(CONFIG_BOARD_STM32F3_DISCO) || \
-	defined(CONFIG_BOARD_NUCLEO_F429ZI) || \
-	defined(CONFIG_BOARD_NUCLEO_F746ZG) || \
-	defined(CONFIG_BOARD_NUCLEO_F767ZI) || \
-	defined(CONFIG_BOARD_NUCLEO_F722ZE) || \
-	defined(CONFIG_BOARD_NUCLEO_G071RB) || \
-	defined(CONFIG_BOARD_NUCLEO_G431RB) || \
-	defined(CONFIG_BOARD_NUCLEO_G474RE) || \
-	defined(CONFIG_BOARD_NUCLEO_H743ZI) || \
-	defined(CONFIG_BOARD_NUCLEO_L073RZ) || \
-	defined(CONFIG_BOARD_NUCLEO_L152RE) || \
-	defined(CONFIG_BOARD_DISCO_L475_IOT1) || \
-	defined(CONFIG_BOARD_NUCLEO_L552ZE_Q) || \
-	defined(CONFIG_BOARD_STM32L562E_DK) || \
-	defined(CONFIG_BOARD_STM32H573I_DK) || \
-	defined(CONFIG_BOARD_B_U585I_IOT02A) || \
-	defined(CONFIG_BOARD_NUCLEO_U575ZI_Q) || \
-	defined(CONFIG_BOARD_NUCLEO_U5A5ZJ_Q) || \
-	defined(CONFIG_BOARD_NUCLEO_WL55JC) || \
+#if defined(CONFIG_BOARD_NUCLEO_F091RC) || defined(CONFIG_BOARD_NUCLEO_F207ZG) ||                  \
+	defined(CONFIG_BOARD_STM32F3_DISCO) || defined(CONFIG_BOARD_NUCLEO_F429ZI) ||              \
+	defined(CONFIG_BOARD_NUCLEO_F746ZG) || defined(CONFIG_BOARD_NUCLEO_F767ZI) ||              \
+	defined(CONFIG_BOARD_NUCLEO_F722ZE) || defined(CONFIG_BOARD_NUCLEO_G071RB) ||              \
+	defined(CONFIG_BOARD_NUCLEO_G431RB) || defined(CONFIG_BOARD_NUCLEO_G474RE) ||              \
+	defined(CONFIG_BOARD_NUCLEO_H743ZI) || defined(CONFIG_BOARD_NUCLEO_L073RZ) ||              \
+	defined(CONFIG_BOARD_NUCLEO_L152RE) || defined(CONFIG_BOARD_DISCO_L475_IOT1) ||            \
+	defined(CONFIG_BOARD_NUCLEO_L552ZE_Q) || defined(CONFIG_BOARD_STM32L562E_DK) ||            \
+	defined(CONFIG_BOARD_STM32H573I_DK) || defined(CONFIG_BOARD_B_U585I_IOT02A) ||             \
+	defined(CONFIG_BOARD_NUCLEO_U575ZI_Q) || defined(CONFIG_BOARD_NUCLEO_U5A5ZJ_Q) ||          \
+	defined(CONFIG_BOARD_NUCLEO_WL55JC) || defined(CONFIG_BOARD_NUCLEO_U083RC) ||              \
```

Comment:
Clang-format is not always right

Please re-instantiate the previous format which was easier to read in this specific case.
For instance it was easy to see which board is added and exactly where it should be added.
With this format, adding one board will require to change all lines

Apply everywhere